plugins/gatsby-plugin-generate-doc-json/utils/handlers.js

```diff
@@ -148,4 +178,6 @@ module.exports = {
   td: (h, node) => h(node, 'td', {}, all(h, node)),
   var: (h, node) => h(node, 'var', {}, [u('text', toString(node))]),
   mark: (h, node) => h(node, 'mark', {}, [u('text', toString(node))]),
+  figcaption: (h, node) =>
+    h(node, 'div', { className: ['meta'] }, all(h, node)),
```

Comment:
Is there any reason this couldn't be a [`figcaption`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption) element instead of a `div`?

Reply:
I guess there is no reason (except for that how's the nr1-help-xp currently handles it). I'm also not super familiar with `figcaption` but I thought it would have to be wrapped by a `figure` element